src/utilities/bcl/RemoteBCL.hpp

```diff
@@ -250,7 +258,7 @@ class UTILITIES_API RemoteBCL : public BCL
   // members
 
   // A helper function to prepare a client, allowing us to change the http_client_config in one place only
-  static web::http::client::http_client getClient(const std::string& url);
+  static web::http::client::http_client getClient(const std::string& url, unsigned timeOutSeconds = 60);
```

Comment:
static getClient  now also takes an optional timeOutSeconds. This one is set to 60 to match prior behavior, even thought I icnreased the RemoteBCL timeout to 120